src/js/burials/components/IntroductionPage.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import { focusElement } from '../../common/utils/helpers';
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+import OMBInfo from '../../common/components/OMBInfo';
+import FormTitle from '../../common/schemaform/FormTitle';
+
+class IntroductionPage extends React.Component {
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+  goForward = () => {
+    this.props.router.push(this.props.route.pageList[1].path);
+  }
+  render() {
+    return (
+      <div className="schemaform-intro">
+        <FormTitle title="Apply online for burial benefits"/>
+        <p>
+          Fill out this application with the most accurate information you have. The more accurate it is, the more likely you are to get a rapid response.
+        </p>
+        <p>
+          VA uses the information you submit to determine your eligibility and to provide you with the best service.
+        </p>
+        <p>
+          Federal law provides criminal penalties, including a fine and/or imprisonment for up to 5 years, for concealing a material fact or making a materially false statement. (See <a href="https://www.justice.gov/usam/criminal-resource-manual-903-false-statements-concealment-18-usc-1001" target="_blank">18 U.S.C. 1001</a>)
+        </p>
+        <div className="usa-alert usa-alert-info">
+          <div className="usa-alert-body">
+            <strong>You won’t be able to save your work or come back to finish.</strong> So before you start, it’s a good idea to gather information about your military and education history, and the school you want to attend.
+          </div>
+        </div>
+        <br/>
+        <div className="row progress-box progress-box-schemaform form-progress-buttons schemaform-buttons">
+          <div className="small-6 usa-width-five-twelfths medium-5 columns">
+            <a href="/healthcare/apply">
+              <button className="usa-button-outline">« Back</button>
+            </a>
+          </div>
+          <div className="small-6 usa-width-five-twelfths medium-5 end columns">
+            <ProgressButton
+                onButtonClick={this.goForward}
+                buttonText="Continue"
+                buttonClass="usa-button-primary"
+                afterText="»"/>
+          </div>
+        </div>
+        <div className="omb-info--container">
+          <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018"/>
```

Comment:
I'm sure this will be changed when we know what to change it to, but can we fill this in with question marks until then to make it obvious?